main.nf

```diff
@@ -440,33 +440,36 @@ process deepTools {
         --plotFileFormat=pdf \\
         --plotTitle="Fingerprints"
 
-    multiBamSummary \\
-        bins \\
-        --binSize=10000 \\
-        --bamfiles $bam \\
-        -out multiBamSummary.npz \\
-        --extendReads=${params.extendReadsLen} \\
-        --ignoreDuplicates \\
-        --centerReads
-
-    plotCorrelation \\
-        -in multiBamSummary.npz \\
-        -o scatterplot_PearsonCorr_multiBamSummary.png \\
-        --corMethod pearson \\
-        --skipZeros \\
-        --removeOutliers \\
-        --plotTitle "Pearson Correlation of Read Counts" \\
-        --whatToPlot scatterplot \\
-
-    plotCorrelation \\
-        -in multiBamSummary.npz \\
-        -o heatmap_SpearmanCorr_multiBamSummary.png \\
-        --corMethod spearman \\
-        --skipZeros \\
-        --plotTitle "Spearman Correlation of Read Counts" \\
-        --whatToPlot heatmap \\
-        --colorMap RdYlBu \\
-        --plotNumbers \\
+    if ((\$(echo "$bam" | wc -w)  > 1));
+    then
+        multiBamSummary \\
+            bins \\
+            --binSize=10000 \\
+            --bamfiles $bam \\
+            -out multiBamSummary.npz \\
+            --extendReads=${params.extendReadsLen} \\
+            --ignoreDuplicates \\
+            --centerReads
+
+        plotCorrelation \\
+            -in multiBamSummary.npz \\
+            -o scatterplot_PearsonCorr_multiBamSummary.png \\
+            --corMethod pearson \\
+            --skipZeros \\
+            --removeOutliers \\
+            --plotTitle "Pearson Correlation of Read Counts" \\
+            --whatToPlot scatterplot
+
+        plotCorrelation \\
+            -in multiBamSummary.npz \\
+            -o heatmap_SpearmanCorr_multiBamSummary.png \\
+            --corMethod spearman \\
+            --skipZeros \\
+            --plotTitle "Spearman Correlation of Read Counts" \\
+            --whatToPlot heatmap \\
+            --colorMap RdYlBu \\
+            --plotNumbers
+    fi
```

Comment:
One small thing, could we get this logic printed to the log? Will make debugging easier. For example:
```bash
else
    echo "Only one BAM input file found. Skipping multiBam commands."
fi
```